docs/faq.md

```diff
@@ -0,0 +1,21 @@
+# FAQ
+
+## When is it appropriate to use Ark instead of etcd's built in backup/restore?
+
+Etcd's restore tooling is good for recovering from data loss in a single etcd cluster, but does not
+support more sophisticated restores. If you only want to recover from data loss in a single etcd
+cluster, you're likely better off using etcd's backup/restore tooling.
+
+Examples of cases where Ark is useful:
+
+* backing up both Kubernetes resources and persistent volume state
+* cluster migrations
+* backing up a subset of your Kubernetes resources
+* backing up Kubernetes resources that are stored across multiple etcd clusters (for example if you
+  run a custom apiserver)
```

Comment:
I wonder if the following might be worthwhile adding to the list too: Ark is also a great option if you don't have access to etcd at all, such as with managed Kubernetes offerings. One example provider where that's the case is Google Container Engine.

(We actually run on GKE and picked Ark for this particular reason (and a few more)).